elliott/elliottlib/cli/find_bugs_bridge_cli.py

```
@@ -611,17 +619,21 @@ def _report_invalid_bugs(self):
         Args:
             mirror_bug: Bridge mirror issue to inspect.
 
         Returns:
-            bool: `True` when the mirror is closed with a Won't Do
-            resolution, otherwise `False`.
+            bool: `True` when the mirror is closed with a terminal resolution,
+            otherwise `False`.
         """
         if mirror_bug.status.lower() != "closed":
             return False
         resolution = (mirror_bug.resolution or "").replace("’", "'").lower()
```

Ruff (0.16.1)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)